node/src/metrics.rs

```diff
@@ -0,0 +1,80 @@
+use once_cell::sync::Lazy;
+pub use prometheus::{
+    self, core::MetricVec, core::MetricVecBuilder, exponential_buckets, linear_buckets, Counter,
+    Encoder, Gauge, GaugeVec, Histogram, HistogramOpts, HistogramVec, IntCounter, IntCounterVec,
+    IntGauge, IntGaugeVec, Opts, Result, TextEncoder,
+};
+
+pub(crate) static NODE_RUNNING: Lazy<IntGaugeVec> = Lazy::new(|| {
```

Comment:
is it by convention to create `Lazy` values for doing metrics in prometheus? If not, we should have it be initialized from one place when the node starts up. It's hard to configure these things down the line since they're globals, but it's not something we have to do right now since there are more important things. If that's the case, then let's also create a ticket to revisit later